docs/release_notes.rst

```diff
@@ -20,6 +20,8 @@ Bug Fixes
   The bug caused some argument names to go unreported in D417 (#448).
 * Fixed an issue where skipping errors on module level docstring via #noqa
   failed when there where more prior comments (#446).
+* Support backslash-continued parameter descriptions in Numpy-style docstrings
```

Comment:
Isn't it doing this for more than just numpy parameter descriptions at this point?

Reply:
should we update this?